src/firejail/profile.c

```diff
@@ -1492,8 +1492,11 @@ int profile_check_line(char *ptr, int lineno, const char *fname) {
 			arg_rlimit_nproc = 1;
 		}
 		else if (strncmp(ptr, "rlimit-fsize ", 13) == 0) {
-			check_unsigned(ptr + 13, "Error: invalid rlimit in profile file: ");
-			sscanf(ptr + 13, "%llu", &cfg.rlimit_fsize);
+			cfg.rlimit_fsize = parse_arg_size(ptr + 13);
+			if ( cfg.rlimit_fsize == 0 ) {
+				perror("Error: invalid rlimit-fsize in profile file. use only non-negative numbers and k,m,g suffix for size");
```

Comment:
suggestion:
 perror("Error: invalid rlimit-fsize in profile file. Only use positive numbers and k,m or g suffix.");